charts/plgd-hub/values.yaml

```diff
@@ -2292,6 +2293,284 @@ certificateauthority:
+  # -- For complete snippet-service service configuration see [plgd/snippet-service](https://github.com/plgd-dev/hub/tree/main/snippet-service)
+  apis:
+    grpc:
+      address:
+      sendMsgSize: 4194304
+      recvMsgSize: 4194304
+      enforcementPolicy:
+        minTime: 5s
+        permitWithoutStream: true
+      keepAlive:
+        # 0s - means infinity
+        maxConnectionIdle: 0s
+        # 0s - means infinity
+        maxConnectionAge: 0s
+        # 0s - means infinity
+        maxConnectionAgeGrace: 0s
+        time: 2h
+        timeout: 20s
+      tls:
+        caPool:
+        keyFile:
+        certFile:
+        clientCertificateRequired: false
+      authorization:
+        ownerClaim:
+        authority:
+        audience:
+        http:
+          maxIdleConns: 16
+          maxConnsPerHost: 32
+          maxIdleConnsPerHost: 16
+          idleConnTimeout: "30s"
+          timeout: "10s"
+          tls:
+            caPool:
+            keyFile:
+            certFile:
+            useSystemCAPool: true
+    http:
+      address:
+      readTimeout: 8s
+      readHeaderTimeout: 4s
+      writeTimeout: 16s
+      idleTimeout: 30s
+  clients:
+    eventBus:
+      subscriptionID: snippet-service
+      nats:
+        url: ""
+        pendingLimits:
+          msgLimit: "524288"
+          bytesLimit: "67108864"
+        tls:
+          caPool:
+          keyFile:
+          certFile:
+          useSystemCAPool: false
+    storage:
+      use: mongoDB
+      mongoDB:
+        uri:
+        database: snippetService
+        maxPoolSize: 16
+        maxConnIdleTime: 4m0s
+        tls:
+          caPool:
+          keyFile:
+          certFile:
+          useSystemCAPool: false
+        bulkWrite:
+          # -- A time limit for write bulk to mongodb. A Timeout of zero means no timeout.
+          timeout: 1m0s
+          # -- The amount of time to wait until a record is written to mongodb. Any records collected during the throttle time will also be written. A throttle time of zero writes immediately. If recordLimit is reached, all records are written immediately
+          throttleTime: 500ms
+          # -- The maximum number of documents to cache before an immediate write.
+          documentLimit: 1000
+      cqlDB:
+        table: snippetServiceRecords
+        hosts: []
+        port: 9142
+        numConnections: 16
+        connectTimeout: 10s
+        # -- Resolve IP address to hostname before validate certificate. If false, the TLS validator will use ip/hostname advertised by the Cassandra node.
+        useHostnameResolution: true
+        reconnectionPolicy:
+          constant:
+            interval: 3s 
+            # 0 - means infinity
+            maxRetries: 3 
+        keyspace:
+          name: plgdhub
+          create: true
+          replication:
+            class: SimpleStrategy
+            replication_factor: 1
+        tls:
+          caPool:
+          keyFile:
+          certFile:
```

Comment:
Do we support cqldb -> If not we need to remove this part for now

Reply:
the configuration exists, but the implementation for cqldb are just empty functions that return error

Reply:
removed